source/working_with_kayobe.rst

```diff
@@ -155,3 +163,57 @@ From the seed host, the Bifrost container may be entered:
    (bifrost-deploy)[root@seed bifrost-base]# baremetal node list
 
 .. Consider adding a section about configuring the physical network.
+
+.. _Kayobe Environments:
+
+Kayobe Environments
```

Comment:
Let's link to the upstream [multiple envs](https://docs.openstack.org/kayobe/latest/multiple-environments.html) docs here as the authoritative source.

Reply:
Done :+1: